packages/portal-proto/src/features/homepage/HPCardImgAlt.tsx

```diff
@@ -1,11 +1,11 @@
 import Link, { LinkProps } from "next/link";
 import { Image } from "@/components/Image";
 
-// TODO eliminate duplicate styles
-const homepageButtonAltStyles = `bg-primary-max text-sm text-summarybar-text border-base-light border-1
-hover:bg-primary-lightest hover:text-primary-min
-font-medium font-heading rounded mt-4 px-4 py-3 w-fit inline-block`;
+const homepageButtonAltStyles = `bg-primary-max text-xl md:text-lg lg:text-sm text-summarybar-link-text border-base-light border-1 hover:bg-primary-lightest hover:text-primary-min font-medium font-heading rounded mt-4 px-4 py-3 inline-block xl:text-sm`;
 
+const imageWrapperStyle = `relative w-[208px] h-[208px] md:w-[165px] md:h-[165px] lg:w-[324px] lg:h-[324px] xl:w-[340px] xl:h-[340px]
```

Comment:
Can we use variables for these pixel value numbers? This style is a little crazy to read as is and the values are repeated on line 34

Reply:
Just fixed this! 👍